CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+_9.6.0_
```

Comment:
Please remove this part. Changelog and version bump are added as part of the release process

Reply:
@mykola-mokhnach  done